recipe/meta.yaml

```diff
@@ -10,33 +10,46 @@ source:
   sha256: b47aecb16771e1ba0736557255d80ae0240b09156bff434321de559b359ac2d6
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - cython >=0.23
-    - numpy
+    - cython
+    - numpy {{ numpy }}
     - pip
     - python
-    - nose
+    - setuptools
+    - wheel
   run:
-    - cython >=0.23
-    - {{ pin_compatible('numpy') }}
     - python
+    - {{ pin_compatible('numpy') }}
 test:
   imports:
     - hdmedians
+  requires:
+    - pip
+    - nose >=1.0
+  source_files:
+    - hdmedians/tests
+  commands:
+    # the pypi package is misconfigured and lists cython and nose as run dependencies.
```

Comment:
There appears to be two conflicting requirement lists:

https://github.com/daleroberts/hdmedians/blob/v0.14.2/setup.py#L17-L18
https://github.com/daleroberts/hdmedians/blob/v0.14.2/pyproject.toml

I'm not sure how this is prioritized, but maybe those lines need to be patched?

Reply:
that works too, I added a patch